mlflow/models/evaluation/base.py

```diff
@@ -1180,16 +1180,15 @@ def evaluate(
     baseline_config=None,
 ):
     '''
-    Evaluate a PyFunc model on the specified dataset using one or more specified ``evaluators``, and
-    log resulting metrics & artifacts to MLflow Tracking. Set thresholds on the generated metrics to
-    validate model quality. For additional overview information, see
-    :ref:`the Model Evaluation documentation <model-evaluation>`.
+    Evaluate a PyFunc model or custom callable on the specified dataset using specified
+    ``evaluators``, and log resulting metrics & artifacts to MLflow tracking server. For additional
```

Comment:
The dataset could also contain the labels, in which case the model / callable doesn't need to be specified. Is there a way to communicate that too?

Reply:
good call! not trivial... will make a followup PR for it.